components/brave_wallet/browser/brave_wallet_pin_service.h

```diff
@@ -18,13 +18,16 @@
 #include "brave/components/brave_wallet/browser/json_rpc_service.h"
 #include "brave/components/brave_wallet/common/brave_wallet.mojom.h"
 #include "brave/components/ipfs/ipfs_service.h"
-#include "brave/components/ipfs/pin/ipfs_local_pin_service.h"
 #include "mojo/public/cpp/bindings/pending_receiver.h"
 #include "mojo/public/cpp/bindings/pending_remote.h"
 #include "mojo/public/cpp/bindings/receiver_set.h"
 #include "mojo/public/cpp/bindings/remote_set.h"
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
+#if BUILDFLAG(ENABLE_IPFS_LOCAL_NODE)
```

Comment:
Do we need this still since this header is also only built when ENABLE_IPFS_LOCAL_NODE is true?